### ISSUE TITLE
Frontend

### ISSUE THREAD

**Issue by Mantics-68** [randyrossi/bmc64#184]
can you use the THEC64MAXI frontend?

**Comment by Kugelblitz360** [comment]
No, that is closed source unfortunately as Retro Games has not provided the source for that. They only open source there changes to VICE and even those not completely.

https://vice-emu.pokefinder.org/index.php/Hall_of_Shame#THE64

**Comment by aminch** [comment]
This is out of [scope](https://github.com/randyrossi/bmc64/blob/master/docs/SCOPE.md) of the project
